Zilon.Core/Zilon.Core/Persons/HumanPerson.cs

```diff
@@ -40,828 +25,11 @@ public class HumanPerson : PersonBase
 
         public override PhysicalSize PhysicalSize { get => PhysicalSize.Size1; }
 
-        public HumanPerson([NotNull] IPersonScheme scheme,
-            [NotNull] ITacticalActScheme defaultActScheme,
-            [NotNull] IEvolutionModule evolutionModule,
-            [NotNull] ISurvivalRandomSource survivalRandomSource) : base()
+        public HumanPerson([NotNull] IPersonScheme scheme) : base()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant 'base()' call.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5480188)